docs/blog/Webots-2018-b-release.md

```diff
@@ -48,7 +48,7 @@ WREN now shines in large worlds, and is still way ahead of OGRE even in simple s
 
 In a bid to make Webots look as realistic as possible, we have implemented a Physically-Based Rendering pipeline into Webots.
 Physically based rendering relies on improved lighting models, and the use of several additional non-colour textures (normal maps, roughness maps and metalness maps) to represent a material's interactions with light to a very high degree of realism.
-This is useable with the new [PBRAppearance](../reference/pbrappearance.md) and [Cubemap](../reference/cubemap.md) nodes.
+This is useable with the new [PBRAppearance](../reference/pbrappearance.md) and `Cubemap` nodes.
 This is still an experimental feature for now, however.
 We will be rolling it out across our objects and simulations as time goes on, and this will likely change the appearance of your simulation worlds.
```

Comment:
No mention about removing the `cubemap` node?

Reply:
This is the blog page for R2018a. This modification is about to not break the link because I dropped the cubemap.md page.

The `Cubemap` node removal is mentioned in the change log: https://github.com/cyberbotics/webots/blob/b1ce3ac41ee9e3b1cf0af77552411f0f9dd3c5bb/docs/reference/changelog-r2020.md

Reply:
Oh yes indeed, I thought this was the changelog.